recipes/wg21-linear_algebra/all/conanfile.py

```diff
@@ -13,12 +16,9 @@ class LAConan(ConanFile):
     license = "NCSA"
     url = "https://github.com/conan-io/conan-center-index"
     settings = "os", "arch", "compiler", "build_type"
+    package_type = "header-library"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def requirements(self):
         self.requires("mdspan/0.1.0")
```

Comment:
Can the library work with newer versions? It's usually better to have the recipes require the newest possible version (0.5.0 in this case), to avoid version conflicts if a user depends on this and another library that have conflicting versions

Reply:
Good point. So yes it can but there has been major refactoring in the Linear Algebra library to pick up the latest changes. As a result, I've removed the previously supported version and updated it to the latest changes in this library (and released 0.7.3 package of the library on the main repo).

There is only one downstream client of this library currently that I am aware of (https://github.com/conan-io/conan-center-index/pull/16252) so I can work with the author to get this updated